### PR TITLE
chore(ci): run solidity tests

### PR DESCRIPTION
**Pull request by ByteNacked**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,3 +69,7 @@ jobs:
           version: nightly
       - name: Run tests
         run: cargo test --release --all-targets
+      - name: Run solidity tests
+        run: |
+          cd ethereum
+          forge test
```
Comment (mertwole) [inline .github/workflows/ci.yml:75]: Maybe as a separate step? It will parallelize workflow run

File: ethereum/test/MessageQueue.t.sol [hunk 2]
```diff
@@ -363,7 +363,8 @@ contract MessageQueueTest is TestHelper {
         );
     }
 
-    function test_submit_transaction() public {
+    // Test skipped, to enable it remove the skip_ prefix
```
Comment (mertwole) [inline ethereum/test/MessageQueue.t.sol:366]: Please also add `TODO` to find this comment later

File: ethereum/test/Treasury.t.sol [hunk 3]
```diff
@@ -33,7 +33,8 @@ contract TreasuryTest is TestHelper {
         treasury.deposit(address(erc20_token), amount, VARA_ADDRESS_3);
     }
 
-    function test_withdraw() public {
+    // Test skipped, to enable it remove the skip_ prefix
```
Comment (mertwole) [inline ethereum/test/Treasury.t.sol:36]: Please also add `TODO` to find this comment later

**Review by mertwole — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ByteNacked) [reply .github/workflows/ci.yml:75]: It runs instantaneously, so not sure if it worth it.

File: ethereum/test/MessageQueue.t.sol (hunk 2 above)
Comment (ByteNacked) [reply ethereum/test/MessageQueue.t.sol]: Fixed

File: ethereum/test/Treasury.t.sol (hunk 3 above)
Comment (ByteNacked) [reply ethereum/test/Treasury.t.sol]: Fixed

**Pull request merged**